be/src/common/config.h

```diff
@@ -1079,6 +1079,9 @@ CONF_mInt32(primary_key_limit_size, "128");
 // used for control the max memory cost when batch get pk index in each tablet
 CONF_mInt64(primary_key_batch_get_index_memory_limit, "104857600"); // 100MB
 
+// used for control the max columns that handle at once when apply
+CONF_mInt32(primary_key_batch_apply_column_limit, "5");
```

Comment:
It seems the same use with `vertical_compaction_max_columns_per_group`.
`vertical_compaction_max_columns_per_group` is used to compact by column group.
I think you can reuse it.

Reply:
Ok